windows/scripts/BinplaceCudaDependencies.cmd

```diff
@@ -0,0 +1,22 @@
+set CUDA_TOOLKIT_BIN_DIR=%~1%
+set CUDNN_PATH=%~2%
+set IS_CPU_ONLY_BUILD=%3%
+set USE_CUDNN=%4%
+set OUTPUT_DIR=%~5%
+
+if %IS_CPU_ONLY_BUILD% == true (
+    echo BinplaceCudaDependencies : CPU only build, don't copy cuda dependencies.
+ ) else (
+    echo BinplaceCudaDependencies : Copy cudart*.dll, cublas*dll, curand*.dll to output.
```

Comment:
It's more standard to require CUDA installed on the target machines. No toolkit that I know ships with CUDA runtime. Caffe Linux build also doesn't copy these files.

What about cuDNN?

I suggest not running this script in the pre-build. We can still leave the script for those who want to run on demand.